airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -20,12 +20,12 @@ deepdiff = "^6.3.0"
 mergedeep = "^1.3.4"
 pydash = "^6.0.2"
 dpath = "^2.1.5"
-dagster-cloud = "^1.2.6"
+dagster-cloud = "^1.5.14"
 grpcio = "^1.47.0"
 poetry2setup = "^1.1.0"
 poetry = "^1.5.1"
-pydantic = "^1.10.6"
-dagster-slack = "^0.20.2"
+pydantic = "^1.10.8"
```

Comment:
I think (hope) we can remove this specific pin and let the dagster-cloud dependency take care of picking the right version for it, similar to what you've done in airbyte ci.